src/xrGame/ai/monsters/poltergeist/poltergeist_telekinesis.cpp

```diff
@@ -84,6 +84,7 @@ void CPolterTele::update_schedule()
         m_time = time();
         m_time_next = 0;
         m_state = eFireObjects;
+        break;
```

Comment:
Are you sure you want to wait for the next update iteration before executing fire? It looks like (m_time + m_time_next < time()) potentially can turn to 'false' on the next update iteration because we don't know the exact time of this iteration.

Reply:
Thanks for the description. I seldom change state and execute it immediately so I add `break` in here. I agree with you that once the object is raised then fire it, will recover it in next commit. Thanks again!

Reply:
Thanks for the description. I seldom change state and execute it immediately so I add `break` in here. I agree with you that once the object is raised then fire it, will recover it in next commit. Thanks again!

Reply:
```suggestion
```